baracoda/sql/schema.sql

```diff
@@ -1,12 +1,36 @@
 DROP SEQUENCE IF EXISTS heron;
-CREATE SEQUENCE heron START 200000;
+CREATE SEQUENCE heron
+START 200000;
 
 DROP TABLE IF EXISTS barcodes;
 
-CREATE TABLE barcodes (
+CREATE TABLE barcodes
+(
     id SERIAL,
     barcode VARCHAR(255) NOT NULL,
     prefix VARCHAR(32) NOT NULL,
     created_at TIMESTAMP NOT NULL,
+    barcodes_group_id integer,
     PRIMARY KEY (id)
-)
+);
+
+DROP TABLE IF EXISTS barcodes_groups;
+CREATE TABLE barcodes_groups
+(
+    id SERIAL,
+    created_at timestamp,
+    PRIMARY KEY (id)
+);
+
+
+DROP TABLE IF EXISTS alembic_version;
```

Comment:
Are we going to have to maintain this table? Or does it insert the new versions numbers after each migration?

Reply:
Alembic needs this table to keep track the versions of database. If we store the schema we need it, and we have to add manually the versions in this schema file to represent what version this schema is on.